subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateSwiftBundle.java

```diff
@@ -39,20 +44,20 @@
 public class CreateSwiftBundle extends DefaultTask {
     private final RegularFileVar informationFile;
     private final RegularFileVar executableFile;
-    private final DirectoryVar outputDir;
+    private final RegularFileVar outputDir;
     private final SwiftStdlibToolLocator swiftStdlibToolLocator;
 
     @Inject
     public CreateSwiftBundle(SwiftStdlibToolLocator swiftStdlibToolLocator) {
         this.informationFile = newInputFile();
         this.executableFile = newInputFile();
-        this.outputDir = newOutputDirectory();
+        this.outputDir = newOutputFile();
         this.swiftStdlibToolLocator = swiftStdlibToolLocator;
```

Comment:
would it make sense for this task to have an `onlyIf { swiftStdlibToolLocator.find() != null }`?

Reply:
I think you are right on this one, added a comment here: https://github.com/gradle/gradle-native/issues/128#issuecomment-332210102.